### DISCUSSION TITLE
BRIDGE_BookClub 4th Book

### DISCUSSION THREAD

**Discussion by fkdl0048** [BRIDGE-DEV/BRIDGE_BookClub#65] [Show and tell]
# BRIDGE_BookClub 4th Book

4회차에 진행할 서적을 미리 추천 받아서 결정하겠읍니다..!

현재 3회차까지 진행된 책은 다음과 같습니다.

- [1회차 클린코드](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3ACleanCode+)
- [2회차 리팩터링](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3ARefactoring+)
- [3회차 클린코더](https://github.com/BRIDGE-DEV/BRIDGE_BookClub/issues?q=label%3A%22The+Clean+Coder%22+)

항상 책을 해당 회차 진행원들과 회의를 통해 결정하기 때문에 이번 3회차에 참여하신 분들은 평소에 읽고 싶었거나 개인적 성장에 도움이 되는 책을 아래 코멘트로 적어주시면 됩니다.

책을 선정하는 기준은 따로 없고 그때 그때 달라지니 편하게 말씀해주셔도 됩니다.

크게 읽는 카테고리를 나눈다면 다음과 같습니다.

- 기본서 (프로그래밍 관련)
- 아키텍처
- 팀빌딩, 애자일 (소프트 스킬)
- 테스트

**Comment by fkdl0048** [comment]
4회차 추천 책 입니다.

- [이득우의 게임 수학](https://www.yes24.com/Product/Goods/107025224)
  - 게임 물리 관련 지식이 부족하다고 생각해서..
- [개발자의 글쓰기](https://www.yes24.com/Product/Goods/79378905)
  - 글쓰기 관련에 도움이 될 것 같은
- [단위 테스트](https://www.yes24.com/Product/Goods/104084175)
  - 테스트코드에 대한 숙련도 및 이해
- [CODE 코드](https://www.yes24.com/Product/Goods/16667186)
  - 하드웨어와 소프트웨어간의 이해
- [실용주의 프로그래머](https://www.yes24.com/Product/Goods/107077663)
  - 개발자 마인드 셋
- [문제 해결을 위한 알고리즘 with 수학](https://www.yes24.com/Product/Goods/122544981)
  - 알고리즘 공부 + 수학
- [클린 아키텍처](https://www.yes24.com/Product/Goods/77283734)
  - 소프트웨어 개발에 필요한
- [알고리즘 문제 해결 전략 세트](https://www.yes24.com/Product/Goods/8006522)
  - 알고리즘 풀이
- [한 권으로 읽는 컴퓨터 구조와 프로그래밍](https://www.yes24.com/Product/Goods/98997716)
  - CS
- [쏙쏙 들어오는 함수형 코딩](https://www.yes24.com/Product/Goods/108748841)
  - 개인적인 호기심
- [테스트 주도 개발](https://www.yes24.com/Product/Goods/12246033)
  - 공부하고 싶은

**Comment by wooyn730** [comment]
저 장바구니에 있는거는 적어보면
- 클린  코드
- 이득우의 게임 수학
- 함께 자라기
- 객체지향의 사실과 오해
이렇게 입니다!

이득우의 게임 수학 읽으면 정말정말 좋을 것 같아요
근데 실습이 많다면 북클럽보단 게임 수학 스터디로 빼는 것도 좋을지도?!

**Comment by luz0415** [comment]
4회차 추천 책입니다!
새로 읽고 싶은 것들..!
- 클린 아키텍처
- 객체지향의 사실과 오해
- 혼자 공부하는 컴퓨터 구조 + 운영체제

읽고 있는데 같이 읽으면 좋겠다고 생각한 것들..!
- 클린 코드
- 게임 프로그래밍 패턴
입니다.  생각보다 겹치는게 많아서 놀랐네요..

**Comment by fkdl0048** [comment]
객체지향과 사실과 오해 결정
